templates/user/profile.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+
+{% block title %}Profile - AcousticBrainz{% endblock %}
+
+{% block content %}
+  <h2>Your profile</h2>
+  <dl class="dl-horizontal">
+    <dt>ID</dt><dd>{{ user.id }}</dd>
```

Comment:
it's not important to show `id` and `created` on this page, but I guess you need at least something here until we put real content up :)